### PR TITLE
Added !guinea command, guinea pigs for everyone!! :D

### PR DESCRIPTION
**Pull request by bheru27**

### CODE REVIEW

File: plugins/guineapigs.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    def match?(cmd)
+        cmd =~ /^(!)?guinea/
+    end
+
+    match /guinea/, method: :guinea, strip_colors: true
+
+    def guinea(m)
+        html = Nokogiri::HTML(open("http://imgur.com/r/guineapigs/").read)
+        img = html.css("a[class=image-list-link]")[Random.rand(59)]['href']
```
Comment (woodruffw) [inline plugins/guineapigs.rb]: `rand(1..59)` is cleaner than `Random.rand`

File: plugins/guineapigs.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        url = "http://imgur.com#{img}" 
+        m.reply url, true
```
Comment (woodruffw) [inline plugins/guineapigs.rb]: This could be consolidated into one line

**Comment by woodruffw** [comment]
COMMANDS.md needs to have !guinea added to it, and version.yml should be bumped by one on the minor number.

File: plugins/guineapigs.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    
+    def usage
+        '!guinea - Get a random guinea pig picture.'
+    end
+
+    def match?(cmd)
+        cmd =~ /^(!)?guinea/
+    end
+
+    match /guinea/, method: :guinea, strip_colors: true
```
Comment (woodruffw) [inline plugins/guineapigs.rb]: `strip_colors` isn't needed here, since we aren't taking an argument

File: plugins/guineapigs.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# guinea.rb
+# Author: Bryan Hernandez
+# -----------------------
+# Display Guinea Pigs images from an imgurl album
+# -----------------------
+#
```
Comment (woodruffw) [inline plugins/guineapigs.rb]: You should list yourself as the lincenser

File: plugins/guineapigs.rb [hunk 5]
```diff
@@ -27,8 +27,7 @@ def match?(cmd)
 
     def guinea(m)
         html = Nokogiri::HTML(open("http://imgur.com/r/guineapigs/").read)
-        img = html.css("a[class=image-list-link]")[Random.rand(59)]['href']
-        url = "http://imgur.com#{img}" 
-        m.reply url, true
+        img = html.css("a[class=image-list-link]")[rand(1..60)]['href']
```
Comment (woodruffw) [inline plugins/guineapigs.rb]: `1..59` is inclusive, you should do `1...60` if you want exclusive

File: plugins/guineapigs.rb [hunk 6]
```diff
@@ -2,7 +2,7 @@
 # guinea.rb
 # Author: Bryan Hernandez
 # -----------------------
-# Display Guinea Pigs images from an imgurl album
+# Display Guinea Pigs images from an imgur album
 # -----------------------
 #
```
Comment (woodruffw) [inline plugins/guineapigs.rb]: License text needed

File: COMMANDS.md [hunk 7]
```diff
@@ -85,7 +85,7 @@ Command Syntax | Plugin class | Description | Working?
 `!xkcd [search]` | XKCDComics | Get a random [XKCD](http://xkcd.com/) comic, or one related to a search. | Yes
 `!yt <search>` | YouTubeSearch | Search [YouTube](https://youtube.com). | Yes
 `!zalgo <text>` | Zalgo | Summon Zalgo with some text. | Yes
-
+`!guinea` | GuineaPigs | Grabs a guinea pig picture from an [imgur album](http://imgur.com/r/guineapigs/). | Yes
```
Comment (woodruffw) [inline COMMANDS.md]: These are alphabetized, should be moved up

**Comment by bheru27** [comment]
Done.

**Comment by woodruffw** [comment]
#24 replaces this.

**Pull request closed without merging**